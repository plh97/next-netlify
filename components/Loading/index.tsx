import React from "react";
import Image from "next/image";
import loadingSvg from "@/assets/loading.svg";

import "./index.scss";

interface IProps {
  text: string;
}

export default function Loading({ text }: IProps) {
  return (
    <div className="Loading">
      <Image alt="loading" src={loadingSvg} />
      {text}
    </div>
  );
}
