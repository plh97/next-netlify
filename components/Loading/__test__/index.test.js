// Link.react.test.js
import React from 'react'
import Loading from '../index.jsx'
import renderer from 'react-test-renderer'

it('renders correctly', () => {
	const tree = renderer.create(<Loading />).toJSON()
	expect(tree).toMatchSnapshot()
})
