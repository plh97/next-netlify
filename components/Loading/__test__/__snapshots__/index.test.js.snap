// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="Loading"
>
  <img
    alt="loading"
    src={Object {}}
  />
  玩命加载中...
</div>
`;
